cpp/linux-sgx/pthread_changes.diff

```diff
@@ -360,9 +360,6 @@ index df3feeb..f2d2fdb 100644
 +            break;
 +        }
 +        if (!m_thread_blocking || !m_thread_mutex.wait(m_deadlock_timeout)) {
-+            std::cout << "The enclave ran out of TCS slots when calling from a new thread into the enclave. Please either increase "
```

Comment:
If we remove these `cout`, there might be a `iostream` header to be removed from the includes?